pandas/tests/indexes/test_multi.py

```diff
@@ -154,7 +154,7 @@ def assert_matching(actual, expected):
             # as much as possible
             self.assertEqual(len(actual), len(expected))
             for act, exp in zip(actual, expected):
-                act = np.asarray(act)
+                act = np.asarray(act, dtype=np.object_)
```

Comment:
can you show me an example? it is passing now, so unclear why you had to change it.